packages/web-client/config/environment.js

```diff
@@ -82,6 +82,7 @@ module.exports = function (environment) {
       testFlightLink: 'https://testflight.apple.com/join/OgFq1EZ0',
       discordSupportChannelUrl:
         'https://discord.com/channels/584043165066199050/899645340746141806',
+      statusPageUrl: 'https://vandelayindustries1.statuspage.io',
```

Comment:
My own statuspage for testing purposes, will add Cardstack's status page once it's fully set up